package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "body-parser": "~1.0.2",
     "cors": "~2.2.0",
-    "express": "~4.0.0",
+    "express": "~4.15.3",
```

Comment:
In other templates I also updated this ones to latest minor without problems
```
"cors": "~2.8.3",
"request": "2.81.0",
"fh-mbaas-api": "~7.0.10"
```
Might be good to do it (but not mandatory)